src/vs/base/browser/markdownRenderer.ts

```diff
@@ -301,3 +301,73 @@ function getInsaneOptions(options: { readonly isTrusted?: boolean }): InsaneOpti
 	};
 }
 
+export function renderMarkdownAsPlaintext(markdown: IMarkdownString) {
```

Comment:
So this function is meant to strip all the markup from the string? So `# Header` would just be rendered as `Header`?

Just want to check because we also have logic to escape markdown content: https://github.com/microsoft/vscode/blob/983ced6961e9206926fc8e16eb41c5f1461a7273/src/vs/base/common/htmlContent.ts#L49 That would render `# Header` as `# Header`


The change itself looks good. If `renderMarkdownAsPlaintext ` is the right approach, then just add a doc comment on it so its behavior is more clear

Reply:
Add documentation!